src/stories/standalone/UIKit/InputControls/DateInput.story.tsx

```diff
@@ -36,6 +41,7 @@ export const DateInputStory = (): React.ReactElement => {
 			placeholder={text("placeholder", "Please Select Date")}
 			value={selectedDate}
 			onChange={handleChange}
+			onError={handleError}
```

Comment:
The useCallback here is unnecessary, you can just pass onError directly or even write `action("onError")` in-line